macros/AddonSidebar.ejs

```diff
@@ -408,12 +411,14 @@ var text = mdn.localStringMap({
       <details>
         <summary><%=text['Distribution/Publishing']%></summary>
         <ol>
+          <li><a href="<%=baseURL%>Distribution"><%=text['Distribution/Signing']%></a></li>
           <li><a href="<%=baseURL%>Distribution"><%=text['Distribution/Signing']%></a></li>
           <li><a href="<%=baseURL%>WebExtensions/Package_your_extension_"><%=text['WebExtensions/Package_your_extension_']%></a></li>
           <li><a href="<%=baseURL%>Distribution/Submitting_an_add-on"><%=text['Distribution/Submitting_an_add-on']%></a></li>
           <li><a href="<%=baseURL%>Source_Code_Submission"><%=text['Source_Code_Submission']%></a></li>
           <li><a href="<%=baseURL%>Distribution/Resources_for_publishers"><%=text['Distribution/Resources_for_publishers']%></a></li>
           <li><a href="<%=baseURL%>Listing"><%=text['Listing']%></a></li>
+          <li><a href="<%=baseURL%>Distribution/Make_money_from_browser_extensions"><%=text['Distribution/Make_money_from_browser_extensions']%>
```

Comment:
This should close the `<a>` and `<li>` tags

Reply:
fixed in the latest commit